requirements.txt

```diff
@@ -1,4 +1,5 @@
+fedora-messaging
```

Comment:
This should be an optional dependency, there is no need to require it for somebody wanting other schemas.